stage-0-proposals.md

```diff
@@ -53,3 +59,13 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [async-context]: https://github.com/legendecas/proposal-async-context
 [async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-23.md#async-context-updates--for-stage-1
 [string-trim-characters]: https://github.com/Kingwl/proposal-string-trim-characters
+[array-async-from]: https://github.com/js-choi/proposal-array-async-from
+[explicit-this-param]: https://github.com/gilbert/es-explicit-this
+[explicit-this-param-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#syntax-for-explicitly-this-argument-for-stage-1
+[this-arg-reflection]: https://github.com/hax/proposal-function-this
+[this-arg-reflection-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#function-thisargumentexpected-property
+[index-from-end]: https://github.com/hax/proposal-index-from-end
+[index-from-end-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-28.md#index-from-end-syntax
+[json-slashes-hint]: https://gist.github.com/hax/5691ca8acdf9179e63043857cdc3616b
+[serializer]: https://github.com/Jack-Works/proposal-serializer
+[maybe]: https://github.com/futurist/proposal-maybe
```

Comment:
which delegate is the champion of this proposal?

Reply:
As I know, the author himself is the delegate of ByteDance.

Reply:
@futurist can you confirm if this is something you plan to present?

Reply:
This proposal had been presented and discussed in previous JSCIG meetings (which have many chinese TC39 delegates and other people in chinese js community), I remember people in the meeting raised some issues which need to do further research and I don't think this proposal is ready to present in TC39 plenary meeting, but it will be very helpful to list it here so more people could see it and give feedbacks.

Reply:
Those meetings aren’t TC39 meetings, however. If you want feedback beyond the discourse and other unofficial mediums, the TC39 plenary is the best place to seek it.

Reply:
How could we collect feedback on TC39 plenary before "ready for present in plenary"?? And not every delegates will participant the plenary, and there are big community never can attend the plenary.

Reply:
> but still get shot down by a single delegate’s constraints.

This is also the case for  "ready" stage 0, stage 1 or stage 2 proposals :-P

I feel what u describe is the ideal situation , but for many complex proposals (normally a more useful proposal would be more complex), they need long time to be "ready", so they need broader feedbacks (use cases, design options, and possible "delegate constraints", etc.) even in stage 0.